packages/knip/src/plugins/changelogen/index.ts

```diff
@@ -0,0 +1,23 @@
+import type { IsPluginEnabled, Plugin } from '../../types/config.js';
+import { toC12config } from '../../util/plugin-config.js';
+import { hasDependency } from '../../util/plugin.js';
+
+// https://github.com/unjs/changelogen
+
+const title = 'Changelogen';
+
+const enablers = ['changelogen', 'changelogithub'];
```

Comment:
Why is `changelogithub` here?

Maybe it should be a separate plugin? Looking at https://github.com/antfu/changelogithub#configuration, it has `changelogithub.config.{json,ts,js,mjs,cjs}` entry files that we are missing here.

Reply:
I added a separate Changelogen plugin. It also uses c12 to load the config:
https://github.com/unjs/changelogen/blob/main/src/config.ts#L2